integrations/attachment_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2020 The Gitea Authors. All rights reserved.
```

Reply:
The file existed before this PR but git think it is a new file even if some content is from the previous file (maybe with not the same indent)

Reply:
checked the diff ... OK -> only was added TestGetAttachment

and on `TestCreateIssueAttachment` the postData

Reply:
@sapk you can resolve this ;)